.github/workflows/version.yml

```diff
@@ -43,10 +43,35 @@ jobs:
             - name: Push tags
               run: git push --tags
 
+            # This Slack step is duplicated in all workflows, if you make a change to this step, make sure to update all
+            # the other workflows with the same change
+            - uses: 8398a7/action-slack@v3
+              name: Job failed Slack notification
+              if: ${{ failure() }}
+              with:
+                  status: custom
+                  fields: workflow, repo
+                  custom_payload: |
+                        {
+                          channel: '#announce',
+                          attachments: [{
+                            color: "#DB4545",
+                            pretext: `<!here>`,
+                            text: `💥 ${process.env.AS_REPO} failed on ${process.env.AS_WORKFLOW} workflow 💥`,
+                          }]
+                        }
+              env:
+                  GITHUB_TOKEN: ${{ github.token }}
+                  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK }}
+
     create_pull_request:
         runs-on: ubuntu-latest
         needs: version
         steps:
+            - uses: actions/checkout@5a4ac9002d0be2fb38bd78e4b4dbde5606d7042f
+              with:
+                  fetch-depth: 0
```

Comment:
Do we need the entire repo or just the branch? https://github.com/actions/checkout#checkout-a-different-branch

Reply:
I'm not sure, we might only need the feature branch and master

Reply:
I think since we probably need those two branches, the simplest for now is just to fetch everything. We can optimize for speed later by doing something more custom like one of the solutions or proposals from https://github.com/actions/checkout/issues/214

Reply:
Let's fetch all for now 👍